src/rdb_protocol/query_server.hpp

```diff
@@ -4,6 +4,8 @@
 
 #include <set>
 
+#include <openssl/ssl.h>
```

Comment:
Same here about include ordering (see point 5 of the style guide https://github.com/rethinkdb/rethinkdb/blob/daniel_coding_style/STYLE.md . This counts as a C system header.)